include/rsp.inc

```diff
@@ -397,52 +397,52 @@ makeOpInstruction vsubc, 0b010101
 makeOpInstruction vxor, 0b101100
 
 /** @brief Load Byte into Vector Register */
-makeLoadInstruction lbv, 0b00000
+makeLoadInstruction lbv, 0b00000, 0
 /** @brief Load Double into Vector Register */
-makeLoadInstruction ldv, 0b00011
+makeLoadInstruction ldv, 0b00011, 3
 /** @brief Load Packed Fourth into Vector Register */
-makeLoadInstruction lfv, 0b01001
+makeLoadInstruction lfv, 0b01001, 4
 /** @brief Load Packed Half into Vector Register */
-makeLoadInstruction lhv, 0b01000
+makeLoadInstruction lhv, 0b01000, 4
 /** @brief Load Long into Vector Register */
-makeLoadInstruction llv, 0b00010
+makeLoadInstruction llv, 0b00010, 2
 /** @brief Load Packed Bytes into Vector Register */
-makeLoadInstruction lpv, 0b00110
+makeLoadInstruction lpv, 0b00110, 3
 /** @brief Load Quad into Vector Register */
-makeLoadInstruction lqv, 0b00100
+makeLoadInstruction lqv, 0b00100, 4
 /** @brief Load Quad (Rest) into Vector Register */
-makeLoadInstruction lrv, 0b00101
+makeLoadInstruction lrv, 0b00101, 4
 /** @brief Load Short into Vector Register */
-makeLoadInstruction lsv, 0b00001
+makeLoadInstruction lsv, 0b00001, 1
 /** @brief Load Transpose into Vector Register */
-makeLoadInstruction ltv, 0b01011
+makeLoadInstruction ltv, 0b01011, 4
 /** @brief Load Unsigned Packed into Vector Register */
-makeLoadInstruction luv, 0b00111
+makeLoadInstruction luv, 0b00111, 3
 
 /** @brief Store Byte from Vector Register */
-makeStoreInstruction sbv, 0b00000
+makeStoreInstruction sbv, 0b00000, 0
 /** @brief Store Double from Vector Register */
-makeStoreInstruction sdv, 0b00011
+makeStoreInstruction sdv, 0b00011, 3
 /** @brief Store Packed Fourth from Vector Register */
-makeStoreInstruction sfv, 0b01001
+makeStoreInstruction sfv, 0b01001, 4
 /** @brief Store Packed Half from Vector Register */
-makeStoreInstruction shv, 0b01000
+makeStoreInstruction shv, 0b01000, 4
 /** @brief Store Long from Vector Register */
-makeStoreInstruction slv, 0b00010
+makeStoreInstruction slv, 0b00010, 2
 /** @brief Store Packed Bytes from Vector Register */
-makeStoreInstruction spv, 0b00110
+makeStoreInstruction spv, 0b00110, 3
 /** @brief Store Quad from Vector Register */
-makeStoreInstruction sqv, 0b00100
+makeStoreInstruction sqv, 0b00100, 4
 /** @brief Store Quad (Rest) from Vector Register */
-makeStoreInstruction srv, 0b00101
+makeStoreInstruction srv, 0b00101, 4
 /** @brief Store Short from Vector Register */
-makeStoreInstruction ssv, 0b00001
+makeStoreInstruction ssv, 0b00001, 1
 /** @brief Store Transpose from Vector Register */
-makeStoreInstruction stv, 0b01011
+makeStoreInstruction stv, 0b01011, 4
 /** @brief Store Unsigned Packed from Vector Register */
-makeStoreInstruction suv, 0b00111
+makeStoreInstruction suv, 0b00111, 3
 /** @brief Store Wrapped vector from Vector Register */
-makeStoreInstruction swv, 0b00111
+makeStoreInstruction swv, 0b00111, 4
```

Comment:
I think I felt it was important because I originally wrote the microcode using armips, which tries to replicate the original syntax closely. Thinking about it, it does make more sense to specify the offset in operand-units instead of bytes because it's a lot less error prone. I will roll this change back and adjust my code (which by the sounds of it would be a lot less work than the alternative).

Reply:
Actually, hold on on this a little bit. I'd like to poll other people.

I think the byte offset is a more natural syntax and also mimics how scalar load/store opcodes work. If I can find a way to automate conversion of my ucode, I think it's not going to be as much as a hassle as I initially thought.